Extensions/permalink_copy.js

```diff
@@ -0,0 +1,124 @@
+//* TITLE permalink_copy **//
```

Comment:
```suggestion
//* TITLE Permalink Copy **//
```
This title is user-facing